lib/foreman_maintain/reporter/cli_reporter.rb

```diff
@@ -271,7 +276,11 @@ def status_label(status)
                     :already_run => { :label => '[ALREADY RUN]', :color => :yellow },
                     :warning => { :label => '[WARNING]', :color => :yellow } }
         properties = mapping[status]
-        @hl.color(properties[:label], properties[:color], :bold)
+        if @plaintext
+          @hl.color(properties[:label], :bold)
```

Comment:
I think plaintext can still have color?

Reply:
The issue with color is that, depending on how you view the log file, you might see the "extra" coloring characters.

Reply:
> The issue with color is that, depending on how you view the log file, you might see the ANSI coloring characters instead of color. I hit that with VIM I think.

then better to keep it colorless!

Reply:
I realized the bold was causing troubles too so I removed it in my latest push.